_summaries/sutskever2014sequence_2.md

```diff
@@ -1,19 +1,79 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{Sutskever2014sequence}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: trowk
+score: 6
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+### What is the core idea?
+
+DNN are models that perform very well on a variety of different tasks. However,
+these models struggle with sequences, problems where the inputs and outputs are
```

Comment:
The motivation is not that they struggle with sequence encoding alone, but a specific sequence-to-sequence problem setting.

Reply:
Fixed this. Updated it to say that "these models struggle with sequence-to-sequence problems, tasks where the inputs
and outputs are not fixed."